src/components/GHListLogo/index.tsx

```diff
@@ -10,6 +18,30 @@ export default function GHListLogo(): JSX.Element {
                     <span className="logoText">Map</span>
                 </div>
             </a>
+            <MapGuideButton
+                id="helpGuide"
+                className="navlink navlink-question-mark"
```

Comment:
It would be better not to define custom classes ad we are using `styled-components`